docs/rst/user_guide/atmosphere/intro.rst

```diff
@@ -96,7 +96,7 @@ Eradiate provides two atmosphere types:
   (:class:`~eradiate.scenes.atmosphere.HomogeneousAtmosphere`): radiative
   properties are uniform within the atmosphere.
 * heterogeneous atmosphere
-  (:class:`~eradiate.scenes.atmosphere.HeterogeneousAtmosphere`): radiative
+  (:class:`~eradiate.scenes.atmosphere.HeterogeneousAtmosphereLegacy`): radiative
```

Comment:
Side note: We'll have to update that when we'll finalise the docs for release.

Reply:
Yes, I agree. I decided to let the content of the tutorials and user guide utilize `HeterogeneousAtmosphereLegacy`, for simplicity.